src/libcollections/vec_map.rs

```diff
@@ -709,6 +764,11 @@ impl<V> DoubleEndedIterator for IntoIter<V> {
     fn next_back(&mut self) -> Option<(uint, V)> { self.iter.next_back() }
 }
 
+#[stable]
+impl<V> ExactSizeIterator for IntoIter<V> {
+    fn len(&self) -> usize { self.len() }
```

Comment:
Nice catch! It should be `self.iter.len()`... But self.iter doesn't implement `ExactSizeIterator` so I'll just remove it